Userland/Libraries/LibWeb/Tests/DOM/Node.js

```diff
@@ -1,62 +1,68 @@
-loadPage("file:///res/html/misc/innertext_textcontent.html");
-
-afterInitialPageLoad(() => {
-    test("Node.textContent", () => {
-        var p = document.getElementsByTagName("p")[0];
-        expect(p.textContent).toBe("This is a very small test page :^)");
-        expect(p.firstChild.textContent).toBe("This is a ");
-        expect(p.firstChild.firstChild).toBe(null);
-
-        p.firstChild.textContent = "foo";
-        expect(p.firstChild.textContent).toBe("foo");
-        expect(p.firstChild.firstChild).toBe(null);
-        expect(p.textContent).toBe("foovery small test page :^)");
-
-        p.textContent = "bar";
-        expect(p.textContent).toBe("bar");
-        expect(p.firstChild.textContent).toBe("bar");
-        expect(p.firstChild.firstChild).toBe(null);
-
-        var p = document.getElementById("source");
-        expect(p.textContent).toBe(`
+describe("Not V8", () => {
```

Comment:
🤔

Reply:
oops, I left that meme in there, turns out